### DISCUSSION TITLE
NocoDB Upgrade [Jan 04]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4806] [Discord Threads]
> 🙋‍♂️ **folk-connect**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### It's automatic. Once you upgrade, downgrade wouldn't be possible. You can take a backup of your existing meta DB file & try out newer version before you make a decision to switch over  
#  
> 🙋‍♂️ **folk-connect**  
#### Ok.. This is what I faced today. I upgraded to 100.2 and then I wanted to downgrade but somehow it wasn't working. So I asked this question. Thanks, I will be more careful while upgrading  
#  
> 🙋‍♂️ **folk-connect**  
#### For storing metadata can we use sqlite. Is it fully supported ?  
#  
> 👨‍🦲 **dstala**  
#### Yes. From 100.2 onwards, we even support column edit on SQLite.  
#  
> 🙋‍♂️ **folk-connect**  
#### So for downgrading just restoring backup of metadata DB and downgrading the code should work right ?  
#  
> 👨‍🦲 **dstala**  
#### yes.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1049168517649403945)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
